articles/connections/enterprise/azure-active-directory/v2/index.md

```diff
@@ -160,4 +160,4 @@ Signing keys are used by the identity provider to sign the authentication token
 
```

Comment:
You should remove the note at the top of the page. Use `version-warning` instead, see https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#user-interface

No `click here`, we want the users to use the dropdown.